auth_admin_passkey/models/res_config.py

```diff
@@ -4,55 +4,50 @@
     def get_default_auth_admin_passkey_send_to_admin(self, fields):
-        icp = self.env['ir.config_parameter']
         return {
-            'auth_admin_passkey_send_to_admin': safe_eval(icp.get_param(
-                'auth_admin_passkey.send_to_admin', 'True')),
+            'auth_admin_passkey_send_to_admin':
+            self.env["ir.config_parameter"].get_param(
+                "auth_admin_passkey.send_to_admin") in ['True', '1'] and 1 or 0
         }
 
+    @api.multi
+    def set_auth_admin_passkey_send_to_admin(self):
+        for config in self:
+            self.env['ir.config_parameter'].set_param(
+                "auth_admin_passkey.send_to_admin",
+                config.auth_admin_passkey_send_to_admin or 'False')
+
     @api.model
     def get_default_auth_admin_passkey_send_to_user(self, fields):
-        icp = self.env['ir.config_parameter']
         return {
-            'auth_admin_passkey_send_to_user': safe_eval(icp.get_param(
-                'auth_admin_passkey.send_to_user', 'True')),
+            'auth_admin_passkey_send_to_user':
+            self.env["ir.config_parameter"].get_param(
+                "auth_admin_passkey.send_to_user") in ['True', '1'] and 1 or 0
         }
 
-    auth_admin_passkey_send_to_admin = fields.Boolean(
-        'Send email to admin user.',
-        help=('When the administrator use his password to login in '
-              'with a different account, Odoo will send an email '
-              'to the admin user.'),
-    )
-    auth_admin_passkey_send_to_user = fields.Boolean(
-        string='Send email to user.',
-        help=('When the administrator use his password to login in '
-              'with a different account, Odoo will send an email '
-              'to the account user.'),
-    )
-
-    @api.multi
-    def set_auth_admin_passkey_send_to_admin(self):
-        self.ensure_one()
-
-        icp = self.env['ir.config_parameter']
-        icp.set_param(
-            'auth_admin_passkey.send_to_admin',
-            repr(self.auth_admin_passkey_send_to_admin))
-
     @api.multi
     def set_auth_admin_passkey_send_to_user(self):
-        self.ensure_one()
+        for config in self:
+            self.env['ir.config_parameter'].set_param(
+                "auth_admin_passkey.send_to_user",
+                config.auth_admin_passkey_send_to_user or 'False')
 
-        icp = self.env['ir.config_parameter']
-        icp.set_param(
-            'auth_admin_passkey.send_to_user',
-            repr(self.auth_admin_passkey_send_to_user))
+    auth_admin_passkey_send_to_admin = fields.Boolean(
+        string='Send email to admin user.',
+        help="""When the administrator use his password to login in """
+             """with a different account, System will send an email """
+             """to the admin user.""")
+
+    auth_admin_passkey_send_to_user = fields.Boolean(
+        string='Send email to user.',
+        help="""When the administrator use his password to login in """
```

Comment:
Hi, 
- you moved a lot of code, that make the diff quite hard to review. I don't see the reason of that ?
- I think adding triple quote is unnecessary.

Other it looks great for me !

Reply:
I agree it is a good idea to get rid of the triple quotes. They were introduced in the fix for 8.0.